test/inductor/test_torchinductor.py

```diff
@@ -8230,6 +8230,28 @@ def fn(pytype, dtype):
                 self.assertEqual(ret_opt, fn(pytype, dtype))
 
 
+if HAS_CPU:
```

Comment:
Is this test CPU specific or could be applied to all devices?

Reply:
Done